packages/angular/src/app/pages/common-profile/common-profile.component.scss

```diff
@@ -0,0 +1,100 @@
+@use "../../../variables.scss" as *;
+@use "sass:math";
+
+:host {
+  width: 100%;
+  display: contents;
+
+  .view-wrapper-scroll {
+    height: 100%;
+  }
+}
+
+$gap-padding: 24px;
+
+profile-card {
```

Comment:
Minor comment.

It's better not to use the tag as a CSS selector
Maybe it will be better to add some class to ```profile-card``` and change this selector?